cmake/modules/FindSaxon.cmake

```diff
@@ -50,11 +50,13 @@ if (JAVA_RUNTIME)
   # Find Saxon 6.5.x
   #
   find_file (SAXON
-    NAMES saxon.jar saxon6.jar saxon-6.5.5.jar saxon-6.5.4.jar saxon-6.5.3.jar
+    NAMES saxon.jar saxon6.jar saxon-6.5.5.jar saxon-6.5.4.jar saxon-6.5.3.jar saxon9he.jar
     PATH_SUFFIXES share/java
                   share/java/saxon
                   share/java/saxon6
-                  share/saxon-6.5/lib 
+                  share/saxon-6.5/lib
+                  java/saxon
+    PATHS ENV HOME
```

Comment:
There is no Saxon package (yet) inder Windows MSYS2, therefore Saxon can be installed at this moment only manually. As we have [discussed earlier ](https://github.com/darktable-org/darktable/pull/1327/files/984a7bd13a423902ffcf4b83bc8e8e88f8b6e46d#diff-8fc11b9e12d17bbdcf6daad21dd61bf4) best is to install Saxon and other manual stuff NOT in a system directory - however then `cmake `wont find them... What would be the best place to install manually, which would cause the least problem at build time?